ledger/catchupaccessor.go

```diff
@@ -847,24 +864,25 @@ func (c *catchpointCatchupAccessorImpl) BuildMerkleTrie(ctx context.Context, pro
 				return nil
 			})
 			if err != nil {
-				break
+				keepWriting = false
+				continue
```

Comment:
This looked like a bug. Should we really continue to process the catchpoint file if there was an error?